packages/jira-adapter/e2e_test/instances/cloud/scriptrunner/settings.ts

```diff
@@ -0,0 +1,51 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import { ElemID, Element, InstanceElement, Values, isInstanceElement } from '@salto-io/adapter-api'
+import _ from 'lodash'
+import { createReference } from '../../../utils'
+import { JIRA, SCRIPT_RUNNER_SETTINGS_TYPE } from '../../../../src/constants'
+
+export type BeforeAfterInstances = {
+  before: InstanceElement
+  after: InstanceElement
+}
+
+const createScriptRunnerSettingsValues = (allElements: Element[]): Values => ({
+  'jql_search_view@b': true,
+  'filter_sync_interval@b': 10,
+  'notifications_group@b': createReference(new ElemID(JIRA, 'Group', 'instance', 'system_administrators@b'), allElements, ['name']),
+  'issue_polling@b': false,
+  'scripts_timezone@b': 'UTC',
+  'excluded_notifications_groups@b': [
+    createReference(new ElemID(JIRA, 'Group', 'instance', 'system_administrators@b'), allElements, ['name']),
+  ],
+  'notify_assignee_reporter@b': false,
+})
+
+export const createScriptRunnerSettingsInstances = (allElements: Element[]): BeforeAfterInstances => {
+  const fetchSettings = allElements
+    .filter(isInstanceElement)
+    .find(instance => instance.elemID.typeName === SCRIPT_RUNNER_SETTINGS_TYPE)
+  if (fetchSettings === undefined) {
+    throw new Error('ScriptRunner settings instance not found')
+  }
+  const before = _.cloneDeep(fetchSettings)
```

Comment:
More efficient to do

```suggestion
  const before = fetchSettings.clone()
```

Reply:
But won't it create problems for the internal objects in the instance elelement? I thought .clone is only one depth copying

Reply:
`.clone()` clones also inner values expects builtin types (`BuiltinTypes.STRING`...) or referenced objects, which is what we need here IIUC